src/coreclr/debug/ee/debugger.cpp

```diff
@@ -1939,7 +1939,7 @@ HRESULT Debugger::Startup(void)
         if (FAILED(hr))
         {
             ShutdownTransport();
-            ThrowHR(hr);
+            return S_OK; // we do not want debugger IPC to block runtime initialization
```

Comment:
Should we make this configurable? This might be hard to track down for customers. It's going to look odd in different tools (some will say this is not a coreclr runtime, other say it's not debuggable). Also, at the very least we should log this to the stress log and return a non-S_OK result to clean up the RC Thread and the Debugger obj. It also makes me wonder why `EnableDiagnostics` isn't earlier or returning a non-s_OK result.